php-nginx/8.5/Dockerfile

```
@@ -0,0 +1,110 @@
+# variables
+    && apk add --no-cache tzdata \
+# forward request and error logs to docker log collector
+    && ln -sf /dev/stdout /var/log/nginx/access.log \
+    && ln -sf /dev/stderr /var/log/nginx/error.log \
+# Bring in curl and ca-certificates to make registering on DNS SD easier
+    && apk add --no-cache curl ca-certificates
+
+RUN rm /etc/nginx/conf.d/default.conf
+
+WORKDIR /app
+EXPOSE "80"
```

Hadolint (2.14.0)

[error] unexpected '"'
expecting '\', A port optionally followed by the protocol, A range of ports optionally followed by the protocol, a new line followed by the next instruction, or at least one space

(DL1000)